packages/kotti-ui/source/kotti-button/KtButton.vue

```diff
@@ -1,21 +1,27 @@
 <template>
-	<button
-		:class="mainClasses"
-		role="button"
-		:type="isSubmit ? 'submit' : 'button'"
-		@click="handleClick"
-	>
-		<i v-if="isLoading" class="kt-circle-loading" />
-		<i v-else-if="icon !== null" class="yoco" v-text="icon" />
-		<span v-if="hasSlot">
-			<slot />
-		</span>
-		<span v-else-if="label !== null" v-text="label" />
-	</button>
+	<div>
```

Comment:
can you add `style="display:contents"`
to avoid any addition of unintentional styles with this div